<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-81U3LVR" name="tracksPageScripts.LoginPageScripts" tests="15" failures="2" timestamp="11 Jun 2018 09:22:22 GMT" time="48.247" errors="0">
  <testcase name="login_tc_1" time="6.333" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_2" time="3.107" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_3" time="2.609" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_4" time="2.700" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_5" time="2.298" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_6" time="2.351" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_7" time="2.478" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_8" time="2.311" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_9" time="2.973" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="login_tc_10" time="3.070" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="forgotPassword_tc_1" time="2.969" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="forgotPassword_tc_2" time="2.756" classname="tracksPageScripts.LoginPageScripts"/>
  <testcase name="forgotPassword_tc_3" time="2.773" classname="tracksPageScripts.LoginPageScripts">
    <failure type="java.lang.AssertionError" message="expected [please enter your email address] but found [Your email is not present in our record.]">
      <![CDATA[java.lang.AssertionError: expected [please enter your email address] but found [Your email is not present in our record.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at tracksPageScripts.LoginPageScripts.forgotPassword_tc_3(LoginPageScripts.java:265)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- forgotPassword_tc_3 -->
  <testcase name="forgotPassword_tc_4" time="2.761" classname="tracksPageScripts.LoginPageScripts">
    <failure type="java.lang.AssertionError" message="expected [please enter valid email address!] but found [Your email is not present in our record.]">
      <![CDATA[java.lang.AssertionError: expected [please enter valid email address!] but found [Your email is not present in our record.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at tracksPageScripts.LoginPageScripts.forgotPassword_tc_4(LoginPageScripts.java:284)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- forgotPassword_tc_4 -->
  <testcase name="login_tc_11" time="6.758" classname="tracksPageScripts.LoginPageScripts"/>
</testsuite> <!-- tracksPageScripts.LoginPageScripts -->
